### ISSUE TITLE
Val accuracy on the potsdam dataset is very low

### ISSUE THREAD

**Issue by kaesar633** [ViTAE-Transformer/LeMeViT#1]
I greatly admire your work, which has made significant advancements in foundational models for remote sensing. I have been working on reproducing the performance of the LeMeVit_tiny model on the Potsdam dataset. Since MillionAid has not released its label files, I pretrained the model on ImageNet, achieving promising results. However, when I load the pretrained weights and fine-tune on the Potsdam dataset, my training accuracy reaches 91%, but the validation accuracy is below 5%. Did you encounter a similar issue during your training process? I would appreciate any insights you could share.

**Comment by kaesar633** [comment]
After my analysis, I found that the issue might be caused by a mismatch between the data labels in /mmsegmentation/mmseg/datasets/potsdam.py in this project and those processed by convert_datasets in mmsegmentation.

**Comment by Winter-Jon** [comment]
Thank you for your interest in and appreciation of our work! I believe the low-precision issue could stem from various factors, such as mismatched learning rate and batch size. I can share our log files with you for reference (there may be some inconsistencies in naming, but they won’t affect). If you could provide more detailed information, such as configs and logs , we might be able to offer additional suggestions.

[20240313_140017.log](https://github.com/user-attachments/files/17971640/20240313_140017.log)
